### PR TITLE
data release

### PR DESCRIPTION
**Pull request by Fructokinase**
frequent_2022_10_25_01_16_38
ipcc_2022_10_03_12_40_31
biomedical_2022_10_21_14_26_25
infrequent_2022_10_24_11_46_31
datcom-store.dc_kg_2022_10_21_12_16_21

### CODE REVIEW

File: internal/server/place/golden/get_locations_ranking/country.json [hunk 1]
```diff
@@ -5,1233 +5,1078 @@
...
-          },
-          {
-            "rank": 232,
-            "value": 18092,
+            "rank": 213,
+            "value": 18174,
             "placeDcid": "country/PLW"
           },
           {
-            "rank": 233,
```
Comment (pradh) [inline internal/server/place/golden/get_locations_ranking/country.json:1167]: @chejennifer also reported this... something to look into

File: internal/server/stat/golden/get_stat_all/result.json [hunk 2]
```diff
@@ -1263,7 +1266,7 @@
               "measurementMethod": "CensusACS1yrSurvey",
               "importName": "CensusACS1YearSurvey",
               "provenanceDomain": "census.gov",
-              "provenanceUrl": "https://www.census.gov/"
+              "provenanceUrl": "https://www.census.gov/programs-surveys/acs/data/data-via-ftp.html"
```
Comment (pradh) [inline internal/server/stat/golden/get_stat_all/result.json:1269]: expected improvement from cl/479678344

File: internal/server/v0/internalplace/golden/get_place_page_data/zuid-nederland.sample.json [hunk 3]
```diff
@@ -569,65 +574,36 @@
...
-            "1980": 12.544945440925,
-            "1981": 11.5781511717945,
-            "1982": 9.38483331924327,
-            "1983": 9.62318002209219,
-            "1984": 10.056391923274,
-            "1985": 10.209256072746,
-            "1986": 9.95697831183292,
-            "1987": 10.4036015729111,
-            "1988": 9.95350625815798,
-            "1989": 10.984029060186,
```
Comment (pradh) [inline internal/server/v0/internalplace/golden/get_place_page_data/zuid-nederland.sample.json:601]: Expected. World Bank did drop some historical data in some cases.

**Review by jehangiramjad — APPROVED** [review]
The WDI-related updates appear to make sense. Approving for WDI

File: internal/server/place/golden/get_locations_ranking/country.json (hunk 1 above)
Comment (chejennifer) [reply internal/server/place/golden/get_locations_ranking/country.json:1167]: it looks like a bunch of countries got removed from the rankings .. is that also expected?

Comment (jehangiramjad) [reply internal/server/place/golden/get_locations_ranking/country.json:1167]: Good point. It appears that the ones that got deleted are not actual "countries" but various territories. I have checked and none of the removed ones appears in the WDI raw data import. I will check to see if they were ever in the older import

Comment (jehangiramjad) [reply internal/server/place/golden/get_locations_ranking/country.json:1167]: Ok spot checking with our last WDI import (raw csv), I don't see these territories in that file either. So this is def not caused by any data missing in the WDI refresh process.

Comment (Fructokinase) [reply internal/server/place/golden/get_locations_ranking/country.json]: This issue is now solved thanks to @chejennifer's fix from https://github.com/datacommonsorg/mixer/pull/953.

**Review by pradh — APPROVED** [review]

**Pull request merged**